<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>142</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>142</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>142</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>懒人听书</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/resource.qrc">
    <normaloff>:/icon/audiobook2.svg</normaloff>:/icon/audiobook2.svg</iconset>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item alignment="Qt::AlignHCenter|Qt::AlignBottom">
      <widget class="QSlider" name="verticalSlider">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_5">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>volume</string>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color:rgba(255,255,255,0);      
color:rgb(0,0,0);                           
border-radius:15px;                          
border: 3px outset rgba(255,255,255,0);         
font:bold 10px;                              
}
 

QPushButton:hover{
background-color:rgba(150,150,150,150);
color:rgb(0,255，255);
}

QPushButton:pressed{
background-color:rgba(150,150,150,200);
color:rgb(0,255，255);
}
 
 
 </string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources/resource.qrc">
         <normaloff>:/icon/volume-notice.svg</normaloff>:/icon/volume-notice.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item alignment="Qt::AlignHCenter|Qt::AlignBottom">
      <widget class="QSlider" name="verticalSlider_2">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="maximum">
        <number>300</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>speed</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color:rgba(255,255,255,0);      
color:rgb(0,0,0);                           
border-radius:15px;                          
border: 3px outset rgba(255,255,255,0);         
font:bold 10px;                              
}
 

QPushButton:hover{
background-color:rgba(150,150,150,150);
color:rgb(0,255，255);
}

QPushButton:pressed{
background-color:rgba(150,150,150,200);
color:rgb(0,255，255);
}
 
 
 </string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources/resource.qrc">
         <normaloff>:/icon/speed (2).svg</normaloff>:/icon/speed (2).svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item alignment="Qt::AlignBottom">
      <widget class="QPushButton" name="pushButton_2">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>Dubbing Type</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color:rgba(255,255,255,0);      
color:rgb(0,0,0);                           
border-radius:15px;                          
border: 3px outset rgba(255,255,255,0);         
font:bold 10px;                              
}
 

QPushButton:hover{
background-color:rgba(150,150,150,150);
color:rgb(0,255，255);
}

QPushButton:pressed{
background-color:rgba(150,150,150,200);
color:rgb(0,255，255);
}
 
 
 </string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources/resource.qrc">
         <normaloff>:/icon/setting-two.svg</normaloff>:/icon/setting-two.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignBottom">
      <widget class="QComboBox" name="comboBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>11111111</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox{
  color:#666666;
  font-size:14px;
  padding: 1px 15px 1px 3px;
  border:1px solid rgba(228,228,228,1);
  border-radius:5px 5px 0px 0px;
} 
 下拉按钮QComboBox::drop-down ，可以设置按钮的位置，大小、背景图，边框等
 
  QComboBox::drop-down {
      subcontrol-origin: padding;
      subcontrol-position: top right;
      width: 15px;
      border:none;
  }
箭头图标 QComboBox::down-arrow这个很简单就是把自己箭头加进去就行。
 
  QComboBox::down-arrow {
      image: url(:/res/work/dateDown.png);
  }
下拉列表QComboBox QAbstractItemView，因为QComboBox的view是QAbstractItemView的子类，所以是后代迭代器的写法。
  
 QComboBox QAbstractItemView{
    background:rgba(255,255,255,1);
    border:1px solid rgba(228,228,228,1);
    border-radius:0px 0px 5px 5px;
    font-size:14px;
    outline: 0px;  //去虚线
  }</string>
       </property>
       <property name="iconSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>11</string>
        </property>
        <property name="icon">
         <iconset resource="resources/resource.qrc">
          <normaloff>:/icon/waves-left.svg</normaloff>:/icon/waves-left.svg</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>22</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="3" alignment="Qt::AlignBottom">
    <widget class="QPushButton" name="pushButton_3">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>play</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgba(255,255,255,0);      
color:rgb(0,0,0);                           
border-radius:15px;                          
border: 3px outset rgba(255,255,255,0);         
font:bold 10px;                              
}
 

QPushButton:hover{
background-color:rgba(150,150,150,150);
color:rgb(0,255，255);
}

QPushButton:pressed{
background-color:rgba(150,150,150,200);
color:rgb(0,255，255);
}
 
 
 </string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources/resource.qrc">
       <normaloff>:/icon/play (1).svg</normaloff>:/icon/play (1).svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QTextEdit" name="textEdit"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources/resource.qrc"/>
 </resources>
 <connections/>
</ui>
